src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,119 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents a List of Tasks, that will be saved to/loaded from
+ * when <code>Duke</code> is run.
+ */
+
+
+public class TaskList {
+    private List<Task> ls;
+
+    public TaskList(List<Task> ls) {
+        this.ls = ls;
+    }
+
+    public TaskList() {
+        this.ls = new ArrayList<>();
+    }
+
+    /**
+     * Lists the string representation of all tasks currently in
+     * the list.
+     */
+    public void listTasks() {
```

Comment:
I like how you always use well-read and intuitive names for the variables and methods 👍